src/messages/components/ConversationAddReactionInner.vue

```diff
@@ -17,6 +17,19 @@
 
 <script>
 import EmojiButton from './EmojiButton'
+import emojiList from 'markdown-it-emoji/lib/data/full.json'
+
+function searchEmoji (search) {
+  // clean search by removing colons and setting lowercase
+  let cleanedSearch = search.replace(/:/g, '').toLowerCase()
+  // only considers exact match searches
+  if (Object.keys(emojiList).includes(cleanedSearch)) {
```

Comment:
I would also show partial matches, it makes it a bit more complex but could be a improvement for users.

Some ideas:
- show partial matches exact ones
- only start the search with more than 2 characters (or if there's an exact match, thinking of `+1`)
- limit number of results (20?)
- show the emoji names to users, then they have an idea why the emoji matched the search

What do you think about it?

Reply:
I agree with the first three bullets and will start working on it! 
About showing the names, I think maybe since the names already pop up once the user hovers over the emoji, it might be overkill to display all the names. If you want I could try to explicitly display the name of the emoji that is currently hovered at the bottom of the popover.

Reply:
> it might be overkill to display all the names

Agree, let's try without for now!

I found another challenge: the list from `markdown-it-emoji` contains duplicates that _could_ show up next to each other in the results. For example, searching for `boat` matches `:boat:` and `:sailboat:`, which both show ⛵️. Ideally we would only show one of them.

Reply:
Added some changes to address that!